site/content/help/feature/tas/index.md

```diff
@@ -0,0 +1,46 @@
++++
+title = "TAS"
+description = "Tool Assisted Speedrun"
++++
+This tool allows you to enter precise controller inputs to the game that is near impossible at a human level without missing a beat. 
+You can set the different analog sticks and lock them to certain positions as well as check certain buttons to be pressed and released.
+
+# How to use
+
+To play back TAS scripts on Yuzu, select the folder with scripts in the configuration menu below
+Tools -> Configure TAS. The file itself has normal text format and has to be called `script0-1.txt`
+for controller 1, `script0-2.txt` for controller 2 and so forth (with max. 8 players).
+
+After placing the file at the correct location, it can be read into Yuzu with the (default) hotkey
+`CTRL+F6` (refresh). In the bottom left corner, it will display the amount of frames the script file
+has. Playback can be started or stopped using `CTRL+F5`.
+
+However, for playback to actually work, the correct input device has to be selected: In the Controls
+menu, select TAS from the device list for the controller that the script should be played on.
```

Comment:
```suggestion
This can be done by going to `Emulation > Configure > Controls` menu, and selecting `TAS` from the device list for the controller that the script should be played on.
```

I'm having problems understanding this one as I haven't done nor seen how to do this, please verify if this is correct.